Resources/Prototypes/Loadouts/neck.yml

```diff
@@ -158,3 +158,216 @@
   exclusive: true
   items:
     - ClothingNeckGoldAutismPin
+
+# Bedsheets
+- type: loadout
+  id: LoadoutNeckBedsheetBlack
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBlack
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetBlue
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBlue
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetBrown
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBrown
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetCosmos
+  category: Neck
+  cost: 3
+  exclusive: true
+  items:
+    - BedsheetCosmos
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetGreen
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetGreen
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetGrey
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetGrey
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
+
+- type: loadout
+  id: LoadoutNeckBedsheetOrange
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetOrange
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+         - Logistics
+    - !type:CharacterJobRequirement
+       inverted: true
+       jobs:
+         - HeadOfPersonnel # Need to blacklist HoP and not command so other heads can wear this bedsheet
+
+- type: loadout
+  id: LoadoutNeckBedsheetPurple
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetPurple
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+         - Epistemics
+    - !type:CharacterJobRequirement
+       inverted: true
+       jobs:
+         - HeadOfPersonnel # Need to blacklist HoP and not command so other heads can wear this bedsheet
```

Comment:
Same here actually, Head Of Personnel is in the Command Department, so he can't take this item anyway.

Reply:
Oh I didn't know this! That's interesting to learn! I didn't know that Jobs can be members of more than one department. Fascinating!